OptimizelySDK.Tests/TestSetup.cs

```diff
@@ -1,25 +0,0 @@
-﻿using NUnit.Framework;
```

Comment:
During my temporary update to NUnit3 this file was causing problems for how the latest version interprets attributes.  

I'll restore it since we're staying with NUnit2. Apologies for the oversight.